README.md

```diff
@@ -213,7 +206,9 @@ facet configuration table is available here.
 | isSearchable | Boolean | if you wish to have a search feature on facets |
 | searchPlaceHolder | String | placeholder for the facet search input |
 | textFacetWrapper | String | css class for the facets list |
-
+| viewMore | Booelan | For enabling view more / less functionality for individual facets  |
```

Comment:
lets call this `enableViewMore` to indicate its a boolean value

Reply:
done